install_config/web_console_customization.adoc

```diff
@@ -265,6 +265,16 @@ oauthConfig:
 Relative paths are resolved relative to the master configuration file. You must
 restart the server after changing this configuration.
 
+When there are multiple login providers configured or when the
+link:../install_config/configuring_authentication.html#identity-providers[`*alwaysShowProviderSelection*`]
+option in the *_master-config.yaml_* file is set to *true*, each time a user's
+token to {product-title} expires, the user is presented with this custom page
+(for example, a Terms of Service page) before they can proceed with other tasks.
```

Comment:
instead of putting this example inline here, I might put it in the paragraph below as one of two examples where a custom login provider selection page can be useful